docs/ovs-pipeline.md

```diff
@@ -121,6 +121,58 @@ are the target of the policy, we have scheduled 2 `nginx` Pods on the same
 Node. They received IP addresses 10.10.1.2 and 10.10.1.3 from the Antrea CNI, so
 you will see these addresses show up in the OVS flows.
 
+## Antrea Network Policy CRD Implementation
+
+In addition to the above tables created for K8s NetworkPolicy, Antrea creates
+additional dedicated tables to support the [ClusterNetworkPolicy](network-policy.md) CRD
+([CnpEgressRuleTable] and [CnpIngressRuleTable]]).
+
+Consider the following ClusterNetworkPolicy as an example for the remainder of
+this document.
+
+```yaml
+apiVersion: security.antrea.tanzu.vmware.com/v1alpha1
+kind: ClusterNetworkPolicy
+metadata:
+  name: cnp0
+spec:
+  priority: 10
+  appliedTo:
+    - podSelector:
+        matchLabels:
+          app: server
+  ingress:
+    - action: Drop
+      from:
+        - podSelector:
+            matchLabels:
+              app: notClient
+      ports:
+        - protocol: TCP
+          port: 80
+  egress:
+    - action: Allow
+      to:
+        - podSelector:
+            matchLabels:
+              app: dns
+      ports:
+        - protocol: UDP
+          port: 53
+```
+
+This CNP is applied to all Pods with the `app: server` label in all
+Namespaces. For these Pods, it drops TCP traffic on port 80 from all
+Pods which have the `app: notClient` label. In addition to the ingress rules,
+this policy also allows egress UDP traffic on port 53 to all Pods with the
+label `app: dns`. Similar to K8s NetworkPolicy, Antrea will only install OVS
+flows for this CNP on Nodes for which some of the Pods are the target of the
+policy. Thus, we have scheduled three Pods (appServer, appDns, appNotClient)
+on the same Node and they have the following IP addresses:
+- appServer: 40.0.0.6
+- appNotClient: 40.0.0.7
+- appDns: 40.0.0.8
+
 ## Tables
 
 ![OVS pipeline](/docs/assets/ovs-pipeline.svg)
```

Comment:
need to update svg, but @Dyanngg reminded me that the ovs-pipeline was updated for antrea proxy as well

Reply:
updated svg